sklearn/manifold/spectral_embedding_.py

```diff
@@ -235,9 +240,15 @@ def spectral_embedding(adjacency, n_components=8, eigen_solver=None,
     if not _graph_is_connected(adjacency):
         warnings.warn("Graph is not fully connected, spectral embedding"
                       " may not work as expected.")
+    if copy:
+        laplacian, dd = csgraph_laplacian(adjacency, normed=norm_laplacian,
+                                          return_diag=True)
+    else:
+        adjacency, dd = csgraph_laplacian(adjacency, normed=norm_laplacian,
```

Comment:
This trick will not avoid to make a copy if `csgraph_laplacian` is doing a copy of `adjacency`. We need to contribute and port the code of scipy if we want to be able to do anything here.

Reply:
Unfortunatelly, `csgraph_laplacian` is doing some copy. We should modify the code such that operation on `graph` are done inplace.

Reply:
https://github.com/scipy/scipy/pull/15251